duo_test.go:26:47: conversion from int to string yields a string of one rune

Go 1.15 beta 1 on Fedora Rawhide
```
Testing    in: /builddir/build/BUILD/duo_api_golang-982114f7995f2fbee17bca19b206fa03e4cc4a12/_build/src
         PATH: /builddir/build/BUILD/duo_api_golang-982114f7995f2fbee17bca19b206fa03e4cc4a12/_build/bin:/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/sbin
       GOPATH: /builddir/build/BUILD/duo_api_golang-982114f7995f2fbee17bca19b206fa03e4cc4a12/_build:/usr/share/gocode
  GO111MODULE: off
      command: go test -buildmode pie -compiler gc -ldflags "-X github.com/duosecurity/duo_api_golang/version=0 -X github.com/duosecurity/duo_api_golang/version.commit=982114f7995f2fbee17bca19b206fa03e4cc4a12 -extldflags '-Wl,-z,relro -Wl,--as-needed  -Wl,-z,now -specs=/usr/lib/rpm/redhat/redhat-hardened-ld '"
      testing: github.com/duosecurity/duo_api_golang
github.com/duosecurity/duo_api_golang
# github.com/duosecurity/duo_api_golang
./duo_test.go:26:47: conversion from int to string yields a string of one rune
./duo_test.go:243:21: conversion from int to string yields a string of one rune
./duo_test.go:244:30: conversion from int to string yields a string of one rune
./duo_test.go:248:21: conversion from int to string yields a string of one rune
./duo_test.go:249:33: conversion from int to string yields a string of one rune
./duo_test.go:253:27: conversion from Duration (int64) to string yields a string of one rune
./duo_test.go:254:22: conversion from Duration (int64) to string yields a string of one rune
FAIL	github.com/duosecurity/duo_api_golang [build failed]
```

See https://github.com/golang/go/issues/32479

--- a/duo_test.go
+++ b/duo_test.go
@@ -6,6 +6,7 @@ import (
 	"io/ioutil"
 	"net/http"
 	"net/url"
+	"strconv"
 	"strings"
 	"testing"
 	"time"
@@ -23,7 +24,7 @@ func TestCanonicalize(t *testing.T) {
 		"5")
 	params := strings.Split(params_str, "\n")
 	if len(params) != 5 {
-		t.Error("Expected 5 parameters, but got " + string(len(params)))
+		t.Error("Expected 5 parameters, but got " + strconv.Itoa(len(params)))
 	}
 	if params[1] != string("POST") {
 		t.Error("Expected POST, but got " + params[1])
@@ -240,18 +241,18 @@ func assertRateLimitedCall(
 	retriedRequestCount := expectedTotalCalls - 1
 
 	if len(httpClient.actualRequests) != expectedTotalCalls {
-		t.Fatal("Made " + string(len(httpClient.actualRequests)) +
-			" requests instead of " + string(expectedTotalCalls))
+		t.Fatal("Made " + strconv.Itoa(len(httpClient.actualRequests)) +
+			" requests instead of " + strconv.Itoa(expectedTotalCalls))
 	}
 
 	if len(sleepSvc.sleepCalls) != retriedRequestCount {
-		t.Fatal("Made " + string(len(sleepSvc.sleepCalls)) +
-			" sleep calls instead of " + string(retriedRequestCount))
+		t.Fatal("Made " + strconv.Itoa(len(sleepSvc.sleepCalls)) +
+			" sleep calls instead of " + strconv.Itoa(retriedRequestCount))
 	}
 	for i := range expectedSleepDurations {
 		if sleepSvc.sleepCalls[i] != expectedSleepDurations[i] {
-			t.Fatal("Slept for " + string(sleepSvc.sleepCalls[i]) +
-				" instead of " + string(expectedSleepDurations[i]))
+			t.Fatal("Slept for " + sleepSvc.sleepCalls[i].String() +
+				" instead of " + expectedSleepDurations[i].String())
 		}
 	}
 }

